backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json

```diff
@@ -10,5 +10,18 @@
   "GeneralSettings": {
     "FrontendBaseUrl": "https://am.ui.at21.altinn.cloud",
     "Hostname": "at21.altinn.cloud"
+  },
+  "KeyVaultSettings": {
+    "SecretUri": "https://altinn-at21-am-kv.vault.azure.net/"
```

Comment:
Why do we need two KeyVault URIs?

Reply:
Removed the clientsettings config and reusing the keyvaultsettings uri